recipes/_configure_indexes.rb

```diff
@@ -36,6 +36,20 @@
       :index
     end
 
+  daily_mb = hash.delete('_maxDailyDataSizeMB')
```

Comment:
I'm not sure I understand your question.  What do you mean it should include padding?